app/src/main/java/ingsw/group1/findmyphone/fragment/LogFragment.java

```diff
@@ -22,9 +28,12 @@
  *
  * @author Riccardo De Zen.
  */
-public class LogFragment extends Fragment {
+public class LogFragment extends Fragment implements PopupMenu.OnMenuItemClickListener {
 
     private LogManager logManager;
+    private RecyclerView logRecycler;
+    private ImageButton sortButton;
+    private SearchView searchView;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'searchView' could be replaced by a local variable.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5028692)